tests/test_main.cpp

```diff
@@ -220,6 +222,8 @@ struct GodotTestCaseListener : public doctest::IReporter {
 			navigation_server_3d = NavigationServer3DManager::new_default_server();
 			navigation_server_2d = memnew(NavigationServer2D);
 
+			//navigation_mesh_generator = NavigationMeshGeneratorManager::get_singleton()->new_default_server();
```

Comment:
Probably should create the mesh generator here and do so before the navigation servers are initialized (though the order might be fine if there is no longer any dependency between the navigation server and mesh generator).

Reply:
Seems to work when done here.

Reply:
No this does not work, the modules require the singleton to register and the test init chain calls MODULE_INITIALIZATION_LEVEL_SERVERS and MODULE_INITIALIZATION_LEVEL_SCENE too early.